node/src/BaseClient.ts

```diff
@@ -2271,6 +2272,26 @@ export class BaseClient {
         return this.createWritePromise(createXRead(keys_and_ids, options));
     }
 
+    /**
+     * Returns the number of entries in the stream stored at `key`.
+     *
+     * See https://valkey.io/commands/xlen/ for more details.
+     *
+     * @param key - The key of the stream.
+     * @returns The number of entries in the stream. If `key` does not exist, returns `0`.
+     *
+     * @example
+     * ```typescript
+     * const numEntries = await client.xlen("my_stream");
+     * console.log(numEntries); // Output: 2 - "my_stream" contains 2 entries.
+     * ```
+     */
+    public xlen(
+        key: string,
+    ): Promise<Record<string, Record<string, string[][]>>> {
```

Comment:
```suggestion
    ): Promise<number> {
```

Reply:
How it worked in tests?

Reply:
good catch, that was a copy-paste error. Good question! I'm not sure... I think we ran into a similar situation with Python